### ISSUE TITLE
[Feature Request] Transaction should have multiple tags

### ISSUE THREAD

**Issue by MohammadShabib** [jameskokoska/Cashew#466]
I've been using Spendee Plus for over three years, and I'm currently planning to switch to a different app. After extensive research, I've found your app to be the best option. However, there's one crucial feature that’s missing, which is the use of tags.

While titles are helpful, tags would significantly enhance the functionality. For example, in Spendee, tags allow for better organization of transactions. Here’s a scenario:

Let's say I order chicken from Taco Bell via Uber Eats and pay with a Chase credit card. The transaction could look like this:

Title: Taco
Tags: Uber Eats, Taco Bell, Chase Bank
This provides more clarity and makes it easier to filter and manage transactions, a feature offered in apps like Spendee and Budget Flow. I understand it may resemble this issue: https://github.com/jameskokoska/Cashew/issues/107, but the inclusion of tags is essential for my workflow.

I’m trying to find an alternative solution, but relying on the title and subcategories alone just isn’t enough for efficient transaction management.

**Comment by jameskokoska** [comment]
Duplicate of #107

**Comment by MohammadShabib** [comment]
@jameskokoska

I see this issue has been closed, but I feel like the decision may have been predetermined. While trying to work around the lack of tag functionality, I noticed something interesting in the search feature for titles.

It seems the search treats comma-separated titles as individual entries rather than one complete title. For instance, if I have these titles:

A
B
A,B
When I search for "A,B", it returns all three titles ("A", "B", and "A,B") instead of just "A,B". This behavior feels more like a feature than a bug 😄 

One possible solution could be to enhance title functionality. When a user enters a comma (","), auto-suggested could suggest existing titles. This would allow titles to work similarly to tags, currently only the first part of the title is auto-suggested, while the rest still has to be manually typed.

This approach might partially address the need for tags by leveraging title auto-completion after commas. It wouldn't be a full replacement for tags but could improve the current functionality.
